specification/cognitiveservices/data-plane/AnomalyDetector/readme.go.md

```diff
@@ -31,5 +31,5 @@ These settings apply only when `--tag=release_1_0 --go` is specified on the comm
 Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ``` yaml $(tag) == 'release_1_0' && $(go)
-output-folder: $(go-sdk-folder)/services/preview/cognitiveservices/v1.0/$(namespace)
+output-folder: $(go-sdk-folder)/services/stable/cognitiveservices/v1.0/$(namespace)
```

Comment:
Can you please remove the `stable` directory.

Reply:
updated.